src/components/Nav/index.tsx

```diff
@@ -1,67 +1,28 @@
-import { useRef } from "react"
-import { useRouter } from "next/router"
+import { lazy, Suspense, useRef } from "react"
 import { useTranslation } from "next-i18next"
-import { BsTranslate } from "react-icons/bs"
-import { MdBrightness2, MdWbSunny } from "react-icons/md"
-import {
-  Box,
-  Button,
-  Flex,
-  HStack,
-  Icon,
-  MenuButton,
-  Text,
-  useColorModeValue,
-  useDisclosure,
-  useEventListener,
-} from "@chakra-ui/react"
+import { Box, Flex, Show, useDisclosure } from "@chakra-ui/react"
```

Comment:
(Only if below suggestion implemented)

```suggestion
import { Box, Flex, Hide, Show, useDisclosure } from "@chakra-ui/react"
```